keda/templates/manager/minimal-rbac.yaml

```diff
@@ -146,4 +146,51 @@ subjects:
 - kind: ServiceAccount
   name: {{ (.Values.serviceAccount.operator).name | default .Values.serviceAccount.name }}
   namespace: {{ .Release.Namespace }}
+{{- if .Values.serviceAccountTokenCreationRoles }}
```

Comment:
I am not sure minimal-rbac.yaml is the correct place for this

@jkremser could you please suggest the best approach here?

Reply:
@zroubalik @wozniakjan Based off the KEDA meeting on Feb 11, I used https://github.com/kedacore/charts/pull/728 as a reference, but IDK if that's the direction for this PR. I also don't want to take credit for this change so if we are fine with this, maybe we can just merge that PR first.

Reply:
Fixed in https://github.com/kedacore/charts/pull/701/commits/dcb6d9740ac40885b73621e61229bb208f83f71d